[ticket-123] added few new tests

VikaDanilova opened the pull request:
In this PR I created a new project and added a few test cases.

VikaDanilova committed:
[ticket-123] added few new tests

vkachal commented:
@VikaDanilova , thanks for the PR. 
Can you please update readMe file and include info hot to run you test and what do I need to have on my laptop setup in order to run the project. Thank you.

VikaDanilova committed:
Updated readme file

VikaDanilova commented:
> @VikaDanilova , thanks for the PR. Can you please update readMe file and include info hot to run you test and what do I need to have on my laptop setup in order to run the project. Thank you.

Hello @vkachal , I updated readme file and added information about prerequisites and how to run the Tests. Please, let me know if you have any questions. Thank you

VikaDanilova committed:
Added setupNodeEvents function to hide data

VikaDanilova committed:
Added custom command for Login

VikaDanilova committed:
Added some comments and hidden data
